cohere/client.py

```diff
@@ -279,15 +279,27 @@ def _validate_chatlog_override(self, chatlog_override: List[Dict[str, str]]) ->
                     message="chatlog_override must be a list of dicts, each mapping the agent to the message."
                 )
 
-    def embed(self, texts: List[str], model: Optional[str] = None, truncate: Optional[str] = None) -> Embeddings:
+    def embed(
+        self,
+        texts: List[str],
+        model: Optional[str] = None,
+        truncate: Optional[str] = None,
+        compress: Optional[bool] = False,
+        compression_codebook: Optional[str] = "default",
+    ) -> Embeddings:
         """Returns an Embeddings object for the provided texts. Visit https://cohere.ai/embed to learn about embeddings.
 
         Args:
             text (List[str]): A list of strings to embed.
             model (str): (Optional) The model ID to use for embedding the text.
             truncate (str): (Optional) One of NONE|START|END, defaults to END. How the API handles text longer than the maximum token length.
+            compress (bool): (Optional) Whether to compress the embeddings. When True, the compressed_embeddings will be returned as integers in the range [0, 255].
```

Comment:
should we include a link to docs to learn more about compressed embeddings?

Reply:
I think we will update the `embed` docs directly to include these changes and we will have a blog post published soon too that we can add to the URL that exist the docstring `https://cohere.ai/embed` cc: @mahjongmen

Reply:
@mkozakov created this issue here https://github.com/cohere-ai/cohere-python/issues/216 and I'll update this once we have the docs ready on the website, resolving this for now